src/main/webapp/app/entities/lecture.model.ts

```diff
@@ -21,6 +22,31 @@ export class Lecture implements BaseEntity {
     channelName?: string;
     isAtLeastEditor?: boolean;
     isAtLeastInstructor?: boolean;
+    ingested?: IngestionState;
 
-    constructor() {}
+    constructor() {
+        this.ingested = this.checkIngestionState ? this.checkIngestionState() : undefined;
```

Comment:
```suggestion
        this.ingested = this.checkIngestionState;
```
Why do you need the ternary here?